jakarta-package-alignment-mappings/src/main/java/com/palantir/gradle/jakartapackagealignment/VersionMappings.java

```diff
@@ -176,17 +175,21 @@ final class VersionMappings {
 
     private VersionMappings() {}
 
-    static Optional<String> getReplacement(ModuleComponentSelector selector) {
-        String key = selector.getGroup() + ":" + selector.getModule();
+    public static Optional<MavenCoordinate> getReplacement(String group, String name, String version) {
+        String key = group + ":" + name;
         VersionMapping mapping = mappings.get(key);
 
         if (mapping != null) {
-            ComparableVersion requestedVersion = new ComparableVersion(selector.getVersion());
+            ComparableVersion requestedVersion = new ComparableVersion(version);
             if (requestedVersion.compareTo(mapping.getMaxJakartaVersionWithJavaxNamespace()) <= 0) {
                 return Optional.of(mapping.getMappedJavaeeCoord());
             }
         }
 
         return Optional.empty();
     }
+
+    public static Map<String, VersionMapping> getMappings() {
+        return mappings;
+    }
```

Comment:
Perhaps this should return `Collection<VersionMapping>` since the key is an implementation detail for quick lookups? we could `return mappings.values();`

Reply:
👍 done